tests/utils/test_get_first_operation.py

```diff
@@ -3,39 +3,41 @@
 from strawberry.utils.operation import get_first_operation
 
 
-def test_document_without_operation_definition_notes():
+def test_document_without_operation_definition_nodes():
     document = parse(
         """
-        fragment Test on Query {
-            hello
-        }
-    """
+		fragment Test on Query {
+			hello
```

Comment:
why the tabs now? 👀

Reply:
Woops, good spot, that was unintentional. Fixed it :)